Assets/Editor/AddScenesToBuild.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+
+[InitializeOnLoad]
+public class AddScenesToBuild : EditorWindow
+{
+    [MenuItem("QA Tools/Open Core Scene Menu")]
+    static void OpenScene()
+    {
+        EditorSceneManager.OpenScene("Assets/QA/Tests/Core Platform Menu/Core Platforms Menu.unity");
```

Comment:
This exact path is also present on line 37. I Would suggest you put it into a constant e.g.

`private const string corePlatformsMenu = "Assets/QA/Tests/Core Platform Menu/Core Platforms Menu.unity"`

and then reference it from all places used.